shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -602,6 +649,12 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
                 }
                 sendAccessibilityEvent(event);
             }
+            if (object.hasFlag(Flag.IS_LIVE_REGION) && !object.hadFlag(Flag.IS_LIVE_REGION)) {
+                sendAccessibilityEvent(object.id, AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED);
+            } else if (object.hasFlag(Flag.IS_TEXT_FIELD) && object.didChangeLabel()
+                    && mInputFocusedObject != null && mInputFocusedObject.id == object.id) {
+                sendAccessibilityEvent(object.id, AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED);
```

Comment:
Maybe add a comment here explaining why this is here for textfield?

Reply:
Done